scripts/Manifest

```diff
@@ -0,0 +1,8 @@
+KUBERNETES_VERSION=1.15.2
```

Comment:
Latest manifest. This is turned into a template for the server to serve other install scripts.